packages/connect-react/src/components/InternalComponentForm.tsx

```
@@ -23,12 +31,40 @@ export function InternalComponentForm() {
     hideOptionalProps, onSubmit,
   } = formContextProps;
 
+  const [
+    sdkErrors,
+    setSdkErrors,
+  ] = useState<ConfigurablePropAlert[]>([])
+
+  useEffect(() => {
+    if (submitting) setSdkErrors([])
+    else {
+      if (__sdkErrors && __sdkErrors.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)